.github/actions/conn-disrupt-test-setup/action.yaml

```diff
@@ -0,0 +1,21 @@
+name: Cilium Conn Disrupt Setup
+description: Setup cilium connection disruption test
+
+inputs:
+  cilium-cli:
+    required: path
```

Comment:
I guess this should be `required: false`.

Reply:
Yes :grimacing: I'm yet another victim of copilot :sob: